src/types/value/link.ts

```diff
@@ -25,6 +25,7 @@ export type EmptyLinkField<
 > = {
 	link_type: Type | string
 	text?: string
+	variant?: string
```

Comment:
❓ #ask: Does this imply someone can choose a link variant without providing a URL, similar to providing text without a URL? Just asking for my own knowledge; the code looks good.

Reply:
yes, same rationale